tests/homework3.py

```diff
@@ -0,0 +1,41 @@
+import time
+import unittest
+
+from selenium.common.exceptions import NoSuchElementException
+
+
+
+from selenium import webdriver
+
+class Login(unittest.TestCase):
+    def setUp(self) -> None:
+        self.browser = webdriver.Chrome("../browsers/chromedriver")
+        self.browser.get('http://hrm-online.portnov.com/')
+
+    def tearDown(self) -> None:
+        self.browser.quit()
+
+
+    def test_homework3(self):
+        browser = self.browser
+
+        browser.find_element_by_id("txtUsername").send_keys("admin")
+        browser.find_element_by_id("txtPassword").send_keys("password")
+        browser.find_element_by_id("btnLogin").click()
+        time.sleep(3)
+        browser.find_element_by_id("empsearch_supervisor_name").send_keys("Bob Boss")
+        browser.find_element_by_id("searchBtn").click()
+        browser.find_element_by_xpath("//*[@id='resultTable']/tbody/tr[2]/td[3]/a").click()
+
+        try:
```

Comment:
Please use an assert command instead.
Try/except blocks should only be reserved for situations when something is not known but may need to be handled as part of the flow of the test. For example, sometimes the button may be on the right, and sometimes on the left. So we will try to click it using locator A but if it's not there we will use locator B.

To check that something is expected we use assert* commands.

Reply:
Also has been fixed